New design 'shelly' - documentation (#4128)
freesewing/freesewing: markdown/org/docs/designs/shelly/options/bodylength/en.md

---
title: "Body length"
---

Controls how long the shirt is.
   - Up to 80% or so will produce a crop top.

>    - 100% ends at the waistband of most pants, with no overlap.

Reviewer: ```suggestion
   - 100% ends at top of the hip bone.
```
